scripts/update_gamma_table

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env Rscript
+stirling_tail <- function(k) {
+  lgamma(k + 1) - (log(sqrt(2 * pi)) + (k + 0.5) * log(k + 1) - (k + 1))
+}
+
+substitute_template <- function(data, src, dest) {
+  template <- read_lines(src)
+  txt <- glue_whisker(template, data)
+  writeLines(txt, dest)
+}
+
+glue_whisker <- function(template, data) {
+  glue::glue(template, .envir = data, .open = "{{", .close = "}}",
+             .trim = FALSE)
+}
+
+len_f <- 128L
+len_d <- 10L
```

Comment:
Was this 16 before, or has it always been 10?

Reply:
always 10